qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -350,7 +352,7 @@ def populate_interfaces_menu(self):
         """Populate then Interfaces menu with all Python and C++ interfaces"""
         self.interfaces_menu.clear()
         interface_dir = ConfigService['mantidqt.python_interfaces_directory']
-        self.interface_list = self._discover_python_interfaces(interface_dir)
+        self.interface_list = self._discover_python_interfaces(interface_dir, self.PYTHON_GUI_BLACKLIST)
```

Comment:
Why pass this when it is an instance variable on the class?

Reply:
This was because `GUI_BLACKLIST` was a constant inside the method, so I couldn't patch it. Thinking about it again I could now not pass it in and do `patch.object` instead?

Reply:
Yeah that's a better way of doing it, could you do that?